core/sql/cli/Cli.cpp

```diff
@@ -9684,15 +9640,17 @@ Lng32 SQLCLI_LOBcliInterface
 	cliRC = cliInterface->executeImmediate(query);
 
 	currContext.resetSqlParserFlags(0x1);
-
+	
 	if (cliRC < 0)
 	  {
 	    cliInterface->retrieveSQLDiagnostics(myDiags);
-	    
+	    if (myDiags->containsError(-CAT_OBJECT_DOES_NOT_EXIST_IN_TRAFODION))
```

Comment:
Found a related issue with drop schema and ddl transactions. I am going to backout this change and not filter out any errors. It does not help with ddl transactions - the diagnostics not being cleared causes problems during transaction commit. So opened JIRA TRAFODION-1883 and will fix it shortly (as part of a separate pull request) .